web/src/app/routing.module.ts

```diff
@@ -95,6 +95,21 @@ const routes: Routes = [
     component: TermsComponent,
     canActivate: [AuthGuard],
   },
+  {
+    path: `${NavigationService.SURVEY_SEGMENT}/:${NavigationService.SURVEY_ID}/${NavigationService.LOI_SEGMENT}/:${NavigationService.LOI_ID}`,
```

Comment:
Can we use destructuring assignment to make these constants more readable? ie somewhere above `const {SURVEY_SEGMENT, LOI_SEGMENT, ...} = NavigationService;`?